### PR TITLE
Perf

### PR DESCRIPTION
**Pull request by renatoi**
@lingyan

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by renatoi** [comment]
Some context:

Grammar is the lib we use to build the regex to parse atomic classes.

There are currently 3 regex that we're interested:

1. Find valid atomic class names on a text;
2. Parse the class name and make sense of it by storing the parsed data in an object;
3. Parse values inside of atomic classes;

For the the first case, we need a not-simple regex, it needs to include every single token of an atomic class name into the regex, so we only find valid classes.

For the second case, we don't need the regex to include everything. We're only interested in the syntactical structure, something like this:

`[parentclass] [:pseudo] [_] [ClassName] [(value)] [!] [:pseudo] [--breakPoint]`

For 3, the regex is simple and very fast since we're only parsing valid values.

Before I was using the same regex for 1 and 2. Now, I'm using a simple regex for 2, so it's faster. That's the `getSyntax(true)` ===> `true` means simple.

File: src/atomizer.js [hunk 1]
```diff
@@ -141,18 +147,28 @@ Atomizer.prototype.parseConfig = function (config/*:AtomizerConfig*/, options/*:
...
+        if (this.rulesMap.hasOwnProperty(match.atomicSelector)) {
+            ruleIndex = this.rulesMap[match.atomicSelector];
+        } else if (this.helpersMap.hasOwnProperty(match.atomicSelector)) {
+            ruleIndex = this.helpersMap[match.atomicSelector];
+        } else if (this.helpersMap.hasOwnProperty(match.selector)) {
+            ruleIndex = this.helpersMap[match.selector];
+        } else {
+            // not a valid class, no op
+            return;
+        }
```
Comment (lingyan) [inline src/atomizer.js]: Add more docs here would be helpful, like what is an `atomicSelector` and `selector`

**Comment by lingyan** [comment]
Left a comment about doc.  Otherwise, :+1:

**Pull request merged**